app/src/main/java/com/x8bit/bitwarden/ui/tools/feature/generator/GeneratorScreen.kt

```diff
@@ -455,8 +455,12 @@ private fun ColumnScope.PasswordTypeContent(
 
     BitwardenSlider(
         value = passwordTypeState.length,
-        onValueChange = passwordHandlers.onPasswordSliderLengthChange,
-        range = passwordTypeState.computedMinimumLength..passwordTypeState.maxLength,
+        onValueChange = { newValue, isUserInteracting ->
+            if (newValue >= passwordTypeState.computedMinimumLength) {
```

Comment:
Do we need this `if` check?

Reply:
It is preventing this behavior where the if the user pulls the slider back it generates a bunch of pw. The same thing happens now but not as fast. More of an edge case but that's why its there.
https://github.com/user-attachments/assets/60117016-cf84-4a0f-a804-352708dc37e2